test/integration/fixtures/blocks/core__pullquote__deprecated-4.serialized.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:pullquote {"textAlign":"left","className":"has-background has-black-background-color is-style-solid-color","backgroundColor":"black","textColor":"white"} -->
-<figure class="wp-block-pullquote has-text-align-left has-background has-black-background-color is-style-solid-color has-white-color has-black-background-color has-text-color has-background"><blockquote><p>pullquote</p><cite>before block supports</cite></blockquote></figure>
+<!-- wp:pullquote {"textAlign":"left","className":"is-style-solid-color","backgroundColor":"black","textColor":"white"} -->
```

Comment:
Why this needed to change?

Reply:
If you check the difference, you'll notice that the old serialization was actually broken, the classnames were duplicated. the new implementation of custom classname actually fixes an issue here.

Reply:
Yes, it looks correct. `has-background` and `has-black-background-color` is injected by the logic related to their corresponding attributes, so they aren't custom class names.